src/service/metrics_middleware.ts

```diff
@@ -17,13 +15,15 @@ export const metricsMiddleware = (config: MetricsConfig, log: Logger) => {
     config.requestDurationBuckets.join(',')
   );
 
+  const excludeRegExp = /^((?!(render)).)*$/;
```

Comment:
how come these routes are excluded?

Reply:
It's to exclude all routes that don't contain `render`. I'm not sure it's really useful currently but it could be in the future (right now, the only route that is excluded is `GET /`)